scripts/src/primazactl/utils/settings.py

```diff
@@ -0,0 +1,50 @@
+import yaml
+import sys
+from primazactl.utils import logger
+
+dry_run = False
```

Comment:
I'd propose to allow `dry_run` to have the following values:
* `None`: no dry-run, execute the command
* `server`: behave as implemented in this PR, check with the target cluster if the command can be performed
* `client`: won't execute anything, just return. This won't require a running target cluster and can be used for printing the yamls and for validation